variants/sensebox_mcu_esp32s2/pins_arduino.h

```diff
@@ -0,0 +1,145 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID 0x303A
+#define USB_PID 0x81B8
+#define USB_MANUFACTURER "senseBox"
+#define USB_PRODUCT "MCU-S2 ESP32S2"
+#define USB_SERIAL "" // Empty string for MAC adddress
+
+// Default USB FirmwareMSC Settings
+#define USB_FW_MSC_VENDOR_ID "senseBox"        // max 8 chars
+#define USB_FW_MSC_PRODUCT_ID "MCU-S2 ESP32S2" // max 16 chars
+#define USB_FW_MSC_PRODUCT_REVISION "1.00"     // max 4 chars
+#define USB_FW_MSC_VOLUME_NAME "senseBox"      // max 11 chars
+#define USB_FW_MSC_SERIAL_NUMBER 0x00000000
+
+#define EXTERNAL_NUM_INTERRUPTS  46
+#define NUM_DIGITAL_PINS         48
+#define NUM_ANALOG_INPUTS        20
+
+#define analogInputToDigitalPin(p) (((p) < 20) ? (analogChannelToDigitalPin(p)) : -1)
+#define digitalPinToInterrupt(p) (((p) < 48) ? (p) : -1)
+#define digitalPinHasPWM(p) (p < 46)
```

Comment:
This part is no longer needed to be defined in the pins_arduino.h file for each variant.
It is defined globally for each SoC. Please remove it to prevent warnings.